client/src/Components/Home/heplers.js

```diff
@@ -0,0 +1,26 @@
+const filterSkills = (arr, skillQuery) => {
+  return arr.filter(
+    item =>
+      !skillQuery.filter(
+        query => !item.skill.filter(_query => _query.id === query.id).length
+      ).length
+  );
+};
+
+const filterOfferTypes = (arr, offerTypeQuery) => {
+  return arr.filter(
+    item =>
+      !offerTypeQuery.filter(
+        query =>
+          !item.offer_type.filter(_query => _query.id === query.id).length
+      ).length
+  );
+};
+
+const filteredOffers = (value1, value2) => {
+  value1.filter(item => {
+    return value2.filter(_item => item.id === _item.id);
+  });
+};
+
+module.exports = { filterSkills, filterOfferTypes, filteredOffers };
```

Comment:
@Fatmasiam Another way to write the exports is to export each function inline (for example `export const filteredOffers = ( .....` and then you don't need the module.exports at the bottom.